kickstarts/base.ks.erb

```diff
@@ -3,6 +3,7 @@
 logging --level=debug
 
 #version=DEVEL
+text
```

Comment:
@geertj Regardless of if we keep this line or not, can you move it up next to the corresponding comment and isolate the pair of lines with blank lines...Something like this?

```text
# Kickstart file automatically generated by anaconda.

# Run the text based installer to see dependency info in screenshots
text

logging --level=debug
```

That way we can keep it in the file and comment/uncomment when needed.